source/Mission.cpp

```diff
@@ -1196,10 +1196,28 @@ Mission Mission::Instantiate(const PlayerInfo &player, const shared_ptr<Ship> &b
 		else
 			it = result.stopovers.erase(it);
 	}
+	const Planet *const sourcePlanet = player.GetPlanet();
+	auto PickPlanetStrategy = [](const Planet *sourcePlanet) -> function<const Planet *(const LocationFilter &,
```

Comment:
```suggestion
	// Returns a function that fits the situation.
	auto PickPlanetStrategy = [](const Planet *sourcePlanet) -> function<const Planet *(const LocationFilter &,
```

When you do some super complex construct like this, how about a comment for noobs like me who have to think 5 minutes baout what this thing actually does